### ISSUE TITLE
Do not document 'etc.' for non-extensible enum types

### ISSUE THREAD

**Issue by rillig** [microsoft/debug-adapter-protocol#278]
The specification contains two kinds of enum types:
* extensible
* non-extensible

For extensible types, it makes sense to have the word 'etc.' in the documentation.
For non-extensible types like `OutputEvent.body.group`, the 'etc.' in the documentation is confusing.

Only extensible enum types should be documented with 'etc.'.

**Comment by connor4312** [comment]
It looks like only `Module.symbolStatus` and `Source.origin` currently use "etc", both of which are 'extensible' (or, more accurately, free form.) `OutputEvent.group` doesn't seem to mention "etc."

Are there other places you see this that I missed?

**Comment by rillig** [comment]
> `OutputEvent.group` doesn't seem to mention "etc."

[OutputEvent](https://microsoft.github.io/debug-adapter-protocol/specification#Events_Output):
![image](https://user-images.githubusercontent.com/3233724/175695032-b6ea41fa-d96d-4cfb-96ce-b2cce5e4f029.png)

Same structure:
* ModuleEvent.body.reason
* ProcessEvent.body.startMethod
* RunInTerminalRequestArguments.kind
* VariablesArguments.filter
* ColumnDescriptor.type
* Source.presentationHint
* StackFrame.presentationHint
* DataBreakpointAccessType
* SteppingGranularity
* CompletionItemType
* ChecksumAlgorithm
* ExceptionBreakMode

In summary, about half of the 30 'etc.' in the specification are wrong.

I generated the above list by searching for `etc.` and noting down each instance of that word that is not followed by the word `string` in the code block immediately below.

**Comment by connor4312** [comment]
Oh, I was looking at the debugAdapterProtocol.json. That "etc" is added at generation time

https://github.com/microsoft/debug-adapter-protocol/blob/2a6afee7fad7ec233cfd163a56846e0b4b002979/spec-generator/generator.ts#L236

**Comment by weinand** [comment]
With the fix the generator appends `etc.` only for "open" enums (indicated in the spec by a leading underscore character).
